app/controllers/content.js

```diff
@@ -56,7 +56,7 @@ export default Ember.Controller.extend(Analytics, {
     expandedAuthors: true,
     showLicenseText: false,
     fileDownloadURL: '',
-    expandedAbstract: false,
+    expandedAbstract: navigator.userAgent.includes('Prerender'),
```

Comment:
In ember, controllers are singletons. If this property gets changed once on any preprint page, there's a good chance that this value would be remembered, and all future preprints you visit (until next reload) would obey the same setting.

Based on the assumption that the prerendered page won't go around clicking any buttons on the page, this shouldn't affect crawlers, so I'd say this is not a blocker for this ticket. But it may be a small user-facing bug. (confirmed on staging)

For future reference, to control initial state every time the page loads, methods like `setupController` or [resetController](http://emberjs.com/api/classes/Ember.Route.html#method_resetController) can be helpful.